tests/running_the_node_and_interrupt.rs

```diff
@@ -40,7 +40,9 @@ fn running_the_node_works_and_can_be_interrupted() {
 			.spawn()
 			.unwrap();
 
-		thread::sleep(Duration::from_secs(30));
+		// Let it produce three blocks.
+		common::wait_n_finalized_blocks(3, 60).await.unwrap();
```

Comment:
I'd prefer the timeout to be passed more explicitly via `Duration::from_secs`, which can be a const in tests/common

Reply:
I'd say yes and no, blocks are more deterministic in some cases, but then it would have to be _truely_ based on block observed rather the using a timeout.